alibi/explainers/ale.py

```diff
@@ -103,6 +107,9 @@ def explain(self, X: np.ndarray, features: Optional[List[int]] = None, min_bin_p
         min_bin_points
             Minimum number of points each discretized interval should contain to ensure more precise
             ALE estimation.
+        grid_points
+            Custom grid points. Must be a dict where the keys are the names of the features and the values are
+            numpy arrays defining the grid.
```

Comment:
Would be best to add an example in the docstring. Also, we will need to update the documentation notebook `ALE.ipynb`.

Reply:
Also, please add a more specific type, i.e. `Optional[Dict[str, np.ndaray]]`. Also, I think we should refer to features by their position (so `int` instead of `str`) as `feature_names` is optional.